spikeinterface/extractors/bids.py

```diff
@@ -55,35 +55,43 @@ def read_bids_folder(folder_path):
 def _read_probe_group(folder, bids_name, recording_channel_ids):
     probegroup = read_BIDS_probe(folder)
 
-    # make maps between : channel_id	and contact_id
-    # use _channels.tsv
+    # make maps between : channel_id and contact_id using _channels.tsv
     for probe in probegroup.probes:
         channels_file = folder / bids_name.replace('_ephys', '_channels.tsv')
         channels = pd.read_csv(channels_file, sep='\t')
+        # channel_ids are unique
         channel_ids = channels['channel_id'].values.astype('U')
+        # contact ids are not unique
+        # a single contact can be associated with multiple channels, contact_ids can be n/a
         channels['contact_id'][channels['contact_id'].isnull()] = -1
```

Comment:
'contact_id' is supposed to be dtype='U' I would use a str values like 'unconnected'
-1 is more indice style
no ?

Reply:
I changed to `unconnected`